<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parabola</class>
 <widget class="QDialog" name="parabola">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCustomPlot" name="plot" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>531</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="plot_btn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>490</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> ( y + constant)^2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> =  x * constant   +    constant</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p_1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>430</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-15.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-15.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-15.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Parabola of form y^2 =4ax :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Parabola of form y^2 =  -4ax :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> ( y + constant)^2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> =   - x * constant   +    constant</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>490</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-15.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>490</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-15.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>490</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-15.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>520</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> ( x + constant)^2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>530</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> =  y * constant   +    constant</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>550</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-15.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p_8">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>550</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-15.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p_9">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>550</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-15.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>580</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> ( x + constant)^2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>590</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> = -  y * constant   +    constant</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p_10">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>610</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-15.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p_11">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>610</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-15.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p_12">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>610</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-15.000000000000000</double>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>550</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Parabola of form x^2 =  4ay :</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>610</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>610</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Parabola of form x^2 =  -4ay :</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
